### DISCUSSION TITLE
about test-apps\display-test-app

### DISCUSSION THREAD

**Discussion by paprophet** [iTwin/itwinjs-core#3233] [Q&A] [answered]
when i open the web first is ok ,but close then web ,and open then web again ,then web show waiting....

![1](https://user-images.githubusercontent.com/18022720/154646701-9245f0fe-60b8-4499-b471-b94bde313d21.png)

![2](https://user-images.githubusercontent.com/18022720/154646723-a9e6697d-1393-4a47-8432-8233bd9417f0.png)

**Comment by pmconne** [comment, marked as answer]
Fixed by #3100.

**Reply by paprophet** [reply]
thank you!

**Reply by paprophet** [reply]
now i git the code ,after [move the elements]  can  not reflash the frontpage
![屏幕截图 2022-02-23 140819](https://user-images.githubusercontent.com/18022720/155269410-4f5bf00f-6a8a-4896-ab2f-b8d07d76286e.png)

**Reply by pmconne** [reply]
Sorry, I cannot understand your question. Please rephrase.

**Reply by paprophet** [reply]
ok i will  try version 3.0.1
